### ISSUE TITLE
How to do the Project Management?

### ISSUE THREAD

**Issue by Jelson9854** [CS6704-VT/Basics-Workshop#2]
You said in class that the API section was optional. Does that include the Project Management section? If it does, is there a way to do SCRUM meetings solo?

**Comment by Jelson9854** [comment]
I cannot seem to assign anyone to this issue

**Comment by chbrown13** [comment]
Yes, I think you need to be added to the project to assign me. I will reserve time in class on Thursday to do the standup meetings, otherwise will need to be completed outside of class.
